docs/cli.md.template

```diff
@@ -0,0 +1,20 @@
+---
+id: cli
+title: CLI
+---
+
+KaTeX installed [using Node.js package managers](node.md) comes with a built-in CLI
+which can be used to render TeX to HTML from the command line. By default, CLI will
+take the input from `stdin`.
+
+```bash
+npx katex
+```
+
+> Above uses the `npx` command to run the locally installed executable.
+You can execute with the relative path: `./node_modules/.bin/katex`
+
+> To use CLI from local clone, you need to build the project first by
+running `npm run dist`
+
+## Usage
```

Comment:
Would "Options" or "Command-Line Options" be better than "Usage"? (Not sure.)

Reply:
@edemaine Changed to `Options`.